src/Microsoft.OpenApi.Readers/OpenApiDocumentExtensions.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers;
+
+namespace Microsoft.OpenApi
```

Comment:
Should it not be in the `Microsoft.OpenApi.Readers` namespace ?

Reply:
Refer to [this comment](https://github.com/microsoft/OpenAPI.NET/pull/1446#discussion_r1377615854)